include/soci/bind-values.h

```diff
@@ -58,6 +58,8 @@ class use_type_vector: public std::vector<use_type_base *>
 
         use_type_vector &p;
         Indicator &ind;
+    private:
+        use_sequence& operator=(const use_sequence&);
```

Comment:
I think it might be preferable to use a `SOCI_NOT_COPYABLE(classname)` macro because relatively soon we will want to have a deleted (in C++11 sense) assignment operator instead of undefined private one (which is just a poor workaround for the really deleted one) and a macro would make it easier to use different expansions for C++98 and C++11.